html/webappapis/dynamic-markup-insertion/opening-the-input-stream/010.html

```diff
@@ -1,22 +0,0 @@
-<!doctype html>
```

Comment:
Let's expand on the reasons for removing this. Seems to be something like:

- No browser currently passes this
- It's unclear what, if anything, it's testing; there are no interesting asserts
- The title, regarding "salvagability", does not appear connected to the test contents. And we will add better tests for salvagability ourselves.